### ISSUE TITLE
New rule: Every description MUST start with a capital letter and end it with a period (".").

### ISSUE THREAD

**Issue by veronicagg** [Azure/azure-openapi-validator#59]
Please find details at https://github.com/Azure/autorest/issues/1966

**Comment by veronicagg** [comment]
@sptramer is this something you'd like to address on your side?

**Comment by sptramer** [comment]
Yes, please move this issue over to `azure/azure-openapi-refchecker`. We may want to make this a "complete sentences" rule where the regex `\.\s*[A-Z]` must be matched, depending on other style restrictions (is "i.e.", "ex.", "e.g.", "etc." etc. allowed).
